content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -41,32 +48,38 @@ $ helm upgrade istio install/kubernetes/helm/istio --namespace istio-system
 
 You can also use Kubernetes’ rolling update mechanism to upgrade the control plane components. This is suitable for cases when Istio hasn't been installed using Helm.
 
-First, generate the desired Istio control plane yaml file, e.g.
+1. Upgrade all the Istio's CRDs via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the Kube api-server:
 
 {{< text bash >}}
-$ helm template install/kubernetes/helm/istio --name istio \
-    --namespace istio-system > install/kubernetes/istio.yaml
+$ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
 {{< /text >}}
 
-or
+1. Render Istio's core components to a Kubernetes manifest file, e.g.
 
 {{< text bash >}}
 $ helm template install/kubernetes/helm/istio --name istio \
-    --namespace istio-system --set global.mtls.enabled=true > install/kubernetes/istio-auth.yaml
+  --namespace istio-system > $HOME/istio.yaml
+{{< /text >}}
+
+If you want to enable [global mutual TLS](/docs/concepts/security/#mutual-tls-authentication), set `global.mtls.enabled` and `global.controlPlaneSecurityEnabled` to `true` for the last command:
+
+{{< text bash >}}
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+  --set global.mtls.enabled=true --set global.controlPlaneSecurityEnabled=true > $HOME/istio-auth.yaml
 {{< /text >}}
 
 If using Kubernetes versions prior to 1.9, you should add `--set sidecarInjectorWebhook.enabled=false`.
```

Comment:
This text also needs to be indented then.

Reply:
done